src/overmind/analytics/effects.ts

```diff
@@ -24,9 +23,9 @@ export const getCarbonReduction = async (): Promise<EffectResponse<MapData>> =>
         }
         resolve(result)
       })
-      .catch((error) => {
+      .catch((_) => {
```

Comment:
In Javascript if you don't use a variable, just don't declare it.
BTW, this should not pass as it's an unused variable. Can you setup your IntelliJ correclty to reflect eslint rules?